flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/validation/ScalarOperatorsValidationTest.scala

```diff
@@ -43,7 +43,7 @@ class ScalarOperatorsValidationTest extends ScalarOperatorsTestBase {
 
   @Test(expected = classOf[ValidationException])
   def testInvalidStringComparison2(): Unit = {
-    testTableApi("w" > 4.toExpr, "FAIL", "FAIL")
+    testTableApi("w".toExpr > 4.toExpr, "FAIL", "FAIL")
```

Comment:
nit: it would be interesting to understand why this is necessary now? if we do with this regression, I would at least remove the right `toExpr` call